### ISSUE TITLE
Create 'Health check summary' component to be displayed at the bottom of the canvas in storybook

### ISSUE THREAD

**Issue by greenc05** [BBC-archive/psammead#4572]
**Is your feature request related to a problem? Please describe.**
This Issue should be completed after https://github.com/bbc/psammead/issues/4571

WebCore use a component to display a summary of component health, this is displayed at the bottom of the canvas in storybook. Lets create our own version of this component to display a health check summary. 

An example can be seen here of the WebCore component:
https://static.files.bbci.co.uk/core/storybook/index.html?path=/story/docs-components-health-factors-summary--basic

Matadata for the WebCore component can be seen here: 
https://github.com/bbc/web/blob/main/packages/gel/components/promo/metadata.json  

The WebCore component is then used to display a summary of component health for WebCore components at the bottom of the canvas, an example can be seen here:
https://static.files.bbci.co.uk/core/storybook/iframe.html?id=gel-components-section-header--basic&args=&viewMode=story

**Describe the solution you'd like**
- Create 'Health check summary' component to be displayed at the bottom of the canvas in storybook (Our own version of the WebCore 'Health Factors summary' component)
- The component should include 'Basic', 'Progress to done' and 'With no data'
- Spacing, font sizes, colours etc - Follow WebCore component
- Icon colours: Green: #0A7B0A, Replace yellow with: #3f3f42, Red: #D60A3D, Black: #3f3f42
- [Screen reader UX](https://paper.dropbox.com/doc/Health-check-Health-check-summary--BThYW6C_ycKhyhRLbz6HcGnjAg-y8T4oXU2Qe0ZjeQiwwnSE) 
- [Accessibility Acceptance Criteria](https://paper.dropbox.com/doc/Health-check-Health-check-summary-AAC--BTgPGZnrl~p5Fe0NhAfxYfTsAg-zxLRZFPMHGB5SQOtDB6NQ)  
- Text as per below screen shots
- This will require a 2 person developer accessibility swarm

Basic
![](https://paper-attachments.dropbox.com/s_B90994C53B45AE6E03D1EC3B3C5E5BDBA0E6997838FF525AA344E30F6DB81A11_1632934842362_Screenshot+2021-09-29+at+17.50.43.png)

Progress to done
![](https://paper-attachments.dropbox.com/s_B90994C53B45AE6E03D1EC3B3C5E5BDBA0E6997838FF525AA344E30F6DB81A11_1632934767796_Screenshot+2021-09-29+at+17.55.39.png)

With no data
![](https://paper-attachments.dropbox.com/s_B90994C53B45AE6E03D1EC3B3C5E5BDBA0E6997838FF525AA344E30F6DB81A11_1632934803629_Screenshot+2021-09-29+at+17.55.59.png)

**Describe alternatives you've considered**

**Testing notes**
[Tester to complete]

Dev insight: Will there be any potential regression? etc

- [ ] This feature is expected to need manual testing.

**Checklist**
- [ ] (BBC contributors only) This issue follows the [repository use guidelines](https://github.com/bbc/simorgh-infrastructure/blob/latest/documentation/repository-guidelines.md)

**Additional context**
